sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_models.py

```diff
@@ -71,24 +71,22 @@ def __init__(self, **kwargs: Any) -> None:
 
     @classmethod
     def _from_generated(cls, key_value: KeyValue) -> PolymorphicConfigurationSetting:
```

Comment:
Isn't `PolymorphicConfigurationSetting` just `ConfigurationSetting`?

Reply:
`PolymorphicConfigurationSetting` also include two sub types `FeatureFlagConfigurationSetting` and `SecretReferenceConfigurationSetting`

Reply:
Why we need them?

We can just use the base class and then in the future if we introduce new types of ConfigurationSetting, we don't need to modify the `PolymorphicConfigurationSetting`, right?

Reply:
Agree, we can just use `ConfigurationSetting` to avoid maintenance for the polymorphic type updates in the future.